DTS-STN/passport-status
# Add Reusable ErrorPage component, update error pages

## [ADO-1218](https://dev.azure.com/DTS-STN/passport-status/_workitems/edit/1218)
## [ADO-1219](https://dev.azure.com/DTS-STN/passport-status/_workitems/edit/1219)

### Description
This PR adds a reusable `ErrorLayout` component with header & footer to match Canada.ca. I've added meta data using our existing component but that does not include OG or DCMI as those should really be added to the `MetaData` component in another PR.

### What to test for/How to test
1. Pull in branch
2. Type `npm run dev`
3. Navigate to `/404` and ensure the page renders properly. Test at mobile screen width to confirm that the top of page anchor appears and works properly

### Additional Notes
Due to the current official workaround for adding the default locale to the path being done through middleware, the user is automatically redirected to `/en/404`. We don't necessarily want this since the page is meant to be bilingual, similar to the splash page, but the workaround that was used for the splash page doesn't work for the error pages and I'm not entirely sure why. Does next handle these error pages differently? Worth noting that while you can specify at the top of the middleware that if the pathname starts with `/404` to not redirect to `/en/404`, this does work but if you manually navigate to `/foo` for example, you still get redirected to `/en/404`.
